templates/data/meta-info.json

```
@@ -156,16 +156,16 @@
     "desc": "This page is a perfect example of why the First Rule of ARIA (i.e. don't use ARIA) is so relevant, especially for mobile screen reader users."
   },
   "spinner.php": {
-    "title": "Accessible Numeric Fields",
+    "title": "Number Input Spinner",
     "desc": "Many developers think form fields with numeric values must always be coded as <input type=\"number\">.  There are exceptions."
   },
   "status.php": {
     "title": "The ARIA Status Role",
     "desc": "If part of a page is updated (e.g. because of an AJAX call), users of screen readers and other assistive tech need to be informed.  Enter the ARIA role \"status\""
   },
   "switch.php": {
-    "title": "The ARIA Switch Role",
-    "desc": "A control with no native HTML5 equivalent, switches have become popular recently in web and mobile applications, but many are inaccessible to screen reader and keyboard-only users.  Here's how to code them correctly."
+    "title": "The ARIA Switch/Toggle Role",
+    "desc": "A control with no native HTML5 equivalent, switches/Toggles have become popular recently in web and mobile applications, but many are inaccessible to screen reader and keyboard-only users.  Here's how to code them correctly."
```

We should be consistent with casing:

```suggestion
    "desc": "A control with no native HTML5 equivalent, switches/toggles have become popular recently in web and mobile applications, but many are inaccessible to screen reader and keyboard-only users.  Here's how to code them correctly."
```